__tests__/app.test.js

```diff
@@ -22,3 +23,23 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+function readEndpointsFile() {
```

Comment:
Great way of getting this data and making your test dynamic! 
Another option is to directly require in the endpoints.json file :)